### ISSUE TITLE
Parse plugin configuration tags

### ISSUE THREAD

**Issue by lucasoares** [vifraa/gopom#3]
Currently, the plugin configuration is not parsed.

Example:

```
      <plugin>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-maven-plugin</artifactId>
        <configuration>
          <skip>true</skip>
        </configuration>
      </plugin>
    </plugins>
```

The current gopom implementation doesn't let you get configurations for a plugin.

**Comment by vifraa** [comment]
You are correct, we are currently not parsing the configuration even though we should.

Feel free to create a PR with the fix. Unfortunately I will not have time to fix this myself for a couple of weeks.

**Comment by vifraa** [comment]
The fix for this is now available in version v0.2.2 thanks to @akadir
